tests/openwisp2/sample_users/tests.py

```diff
@@ -0,0 +1,36 @@
+from openwisp_users.tests.test_admin import (
+    TestBasicUsersIntegration as BaseTestBasicUsersIntegration,
+    TestMultitenantAdmin as BaseTestMultitenantAdmin,
+    TestUsersAdmin as BaseTestUsersAdmin,
+)
+from openwisp_users.tests.test_models import TestUsers as BaseTestUsers
+
+additional_fields = [
+    ('social_security_number', '123-45-6789'),
+]
+
+
+class TestUsersAdmin(BaseTestUsersAdmin):
+    app_label = 'sample_users'
+    is_integration_test = True
+    _additional_user_fields = additional_fields
```

Comment:
`UserAdmin` is tested here. That's why I removed it

Reply:
I... don't thing this runs the Integration tests as well!

Reply:
ok I understand what you mean. Thank you for double checking. I think @devkapilbansal is right and we do not need to run integration tests for openwisp-users because we'll be running the full admin tests.

Reply:
Okay, good to know. 
Sorry, I remembered incorrectly! :smile: